### DISCUSSION TITLE
Translation to bosnian/serbian/croatian

### DISCUSSION THREAD

**Discussion by asimic94** [KL-Psychological-Methodology/ESMira#12] [Ideas]
Hello,

I apologize for posting on the forum twice. I have spent the past week testing ESMira, and I am extremely impressed by the app's capabilities. It works much better than some premium apps that I have previously used, and it is also more intuitive and without glitches. Thank you for making this amazing tool freely available!

I would like to offer a small contribution to your work. I am willing to help with translations to Bosnian/Serbian/Croatian, which is my mother tongue. However, the process may take longer than expected since I am currently very busy with research and teaching. If you are okay with it, I can follow your instructions and start translating :)

**Comment by JodliDev** [comment]
That would help us a lot! :)
And it is ok if it takes a while. The platform adds your translations automatically so it can happen bit by bit :)

Just one question to prove my cultural insensitivity:
You'd want to translate all three languages? If I am not mistaken, they are quite similar but slightly different so they have to be added as distinct languages to our translation platform. So you would have to do a lot of copy and paste work.
Or would you prefer to just work on one of them for now (if yes, which one?)

**Comment by JodliDev** [comment]
I am in the process of preparing a wiki entry with all informations needed for translators. But for now in a gist:

You will have to create an account on our translation platform first: https://translate.jodli.dev


ESMira has 5 language components that need to be translated:
The web interface: https://translate.jodli.dev/projects/esmira/web/
The Android app: https://translate.jodli.dev/projects/esmira/apps/
The about page of ESMira: https://translate.jodli.dev/projects/esmira/about/
The iOS app: https://translate.jodli.dev/projects/esmira/ios/
Some special strings for iOS: https://translate.jodli.dev/projects/esmira/ios_plurals/


To simplify translation, each string has added an atomatic transtaltion (done by DeepL), which you can just accept by pressing the checkmark above it.
If you could give us feedback on how accurate they are, that would also help us a lot :) 


If you are ok with that, we would add your name to our list of contributors for translations.

**Reply by asimic94** [reply]
Hi, 

I'm very sorry for seeing this reply just now. Yes, I'm fine with your conditions. As soon as I'm ready, I'll start working on the translation!

**Reply by JodliDev** [reply]
No worries. I have added all three languages for now. Tell us if you have any questions, or if any strings are unclear.

Thank you for your help! :)
